R/vdiagram-render.R

```diff
@@ -194,8 +194,21 @@ vdiagramTable <- function(reportObject){
   addKableOpts(df, tableId = "vdiagram-table")
 }
 
-addKableOpts <- function(df, tableId){
+#' Excluded Conditions Message
+#' @description Generates the excluded conditions message to show on the plot 
+#' @param reportObject The full report JSON object
+excludedConditionsMessage <- function(reportObject){
+  excludedConditions <- parseExcludedControlConditions(reportObject)
+  returnString <- createControlConditionsString(excludedConditions)
+  
+  if(nchar(returnString) > 0){
+    returnString <- paste("***Measurements with the following control conditions are excluded:&nbsp;***", returnString)
+  }
```

Comment:
I was just going with excluding the message altogether when no conditions were excluded, but I can ask Laura what she'd prefer